main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
```suggestion
#Loop back to this point once code finishes
```
On lines 2, 5, 12, 22 the incorrect syntax for comments is being used. For Python, we use # for commenting, this looks like JavaScript.